spec/spec_helper.rb

```diff
@@ -5,13 +5,14 @@
 require "bundler"
 Bundler.setup
 
-require 'detect_rails_version'
-ENV['RAILS'] = detect_rails_version
+# Run tests with NOCOVER=true to skip code coverage
+unless ENV["NOCOVER"]
+  # Code coverage libraries must be required before all application / gem / library code.
+  require 'simplecov'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.